src/client/pages/forum/Comment.jsx

```diff
@@ -16,24 +20,18 @@ const Comment = () => {
 					<div className="comment-card-header">
 						<div className="comment-user">
 							<img
-								src="./assets/pokemon3.png"
+								src={comment.user.profile ? comment.user.profile.profilePicture : './assets/default-user.jpg'}
```

Comment:
There is a default profile picture already assigned in the database so this image check isn't needed.

Reply:
ok, Thanks